core/src/main/java/io/questdb/cairo/MetadataCacheWriter.java

```diff
@@ -40,22 +39,13 @@ public interface MetadataCacheWriter extends QuietCloseable {
      */
     void hydrateAllTables();
 
-    /**
-     * Updates the metadata cache using table name. If name is invalid
-     * the method will throw TableReferenceOutOfDateException.
-     *
-     * @param tableName The name of the table to read metadata.
-     * @throws io.questdb.cairo.sql.TableReferenceOutOfDateException if the table name is invalid
-     */
-    void hydrateTable(@NotNull CharSequence tableName) throws TableReferenceOutOfDateException;
-
     /**
      * Updates the metadata cache with new entry for the given table. The entry values are
      * read from the _meta file.
      *
      * @param token The table token for the table to read metadata.
      */
-    void hydrateTable(@NotNull TableToken token);
+    void hydrateTable(@NotNull TableToken token, boolean throwError);
```

Comment:
nic: javadoc is missing param description

Reply:
removed Boolean parameter here: https://github.com/questdb/questdb/pull/5159/commits/6b9f046277283aa0e9249fba857d3b06d7235259